platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/BaseEventHashedData.java

```diff
@@ -330,6 +330,7 @@ public long getSelfParentGen() {
      * @return the maximum generation of the other parents
      * @deprecated this method should be replaced since there can be multiple other parents.
      */
+    @Deprecated
```

Comment:
If this is required to pass, we may be a bit too strict enforcing recommendations.
There can be legitimate reasons to mark methods deprecated in javadoc without also annotating them...